src/assume-role.ts

```diff
@@ -2,6 +2,14 @@
 import aws from 'aws-sdk'
 import { execFileSync } from 'child_process'
 
+/**
+ * Allow Mira to assume a role based on a given arn. This is used for deployment
+ * and allows Mira to use the account specified in the configuration file.
+ *
+ * @internal
+ * @throws Cannot assume role ${roleArn}: Invalid Role
+ * @throws Cannot assume role ${roleArn}: Reason
```

Comment:
```suggestion
 * @throws Cannot assume role ${roleArn}: <other reason>
```
Is that better?

Reply:
I used html syntax and that seemed to work ok: 

```html
&lt;other reason&gt;
```

![image](https://user-images.githubusercontent.com/5331111/90647385-2a0ca200-e230-11ea-947e-bdc49dd93995.png)

Reply:
Oh yes ofc!